src/module.c

```diff
@@ -100,30 +100,44 @@ void ReplyWithAggValue(RedisModuleCtx *ctx, timestamp_t last_agg_timestamp, Aggr
 
 int TSDB_range(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     RedisModule_AutoMemory(ctx);
-    
-    if (argc < 4 || argc > 6) return RedisModule_WrongArity(ctx);
 
     long long start_ts, end_ts;
+    long long time_delta = 0;
+    RedisModuleString * aggTypeStr = NULL;
+
+    int pRes = REDISMODULE_ERR;
+    switch (argc) {
+        case 4:
+            pRes = RMUtil_ParseArgs(argv, argc, 2, "ll", &start_ts, &end_ts);
+            break;
+        /* case 5: return RedisModule_ReplyWithLongLong(ctx, 12341234); */ // Test false positive
```

Comment:
@someburner i think this line can be safely deleted, any reason you kept this?

Reply:
It was to test my test-cases (since I was testing for crash by loading the module, not using the py tests). So yeah can be removed.